src/main/java/juja/sqlcmd/view/Console.java

```diff
@@ -0,0 +1,18 @@
+package juja.sqlcmd.view;
+
+import java.util.Scanner;
+
+public class Console implements View {
```

Comment:
I think this class needs to constructor with input and output streams as parameters